array-manipulation-1/src/tail.js

```diff
@@ -0,0 +1,15 @@
+/* exported tail */
+//  New list containing all items after the first item
+//  Go through every item in the list
+//    IF the item is not the first item, put it in the new list
+//  Give back the new list
+
+function tail(array) {
+  var newArr = [];
+  for (let i = 0; i < array.length; i++) {
+    if (i !== 0) {
```

Comment:
Instead of having a conditional run on every iteration of this loop, you should just start the loop one index later